openapi/src/main/kotlin/com/expediagroup/sdk/generators/openapi/OpenApiSdkGenerator.kt

```diff
@@ -58,11 +61,16 @@ class OpenApiSdkGenerator {
     fun run() {
         try {
             val config = CodegenConfigurator().apply {
+                val createTempFile = Files.createTempFile("", ".yaml")
```

Comment:
(naming): `inputFile`, `configsFile`, or `tempConfigsFile`. It should be a name, not a verb.